vcx/libvcx/src/settings.rs

```diff
@@ -70,12 +74,18 @@ pub fn set_defaults() -> u32 {
     settings.insert(CONFIG_GENESIS_PATH.to_string(), DEFAULT_GENESIS_PATH.to_string());
     settings.insert(CONFIG_WALLET_KEY.to_string(),TEST_WALLET_KEY.to_string());
     settings.insert(CONFIG_LINK_SECRET_ALIAS.to_string(), DEFAULT_LINK_SECRET_ALIAS.to_string());
+    settings.insert(CONFIG_EXPORTED_WALLET_PATH.to_string(), DEFAULT_EXPORTED_WALLET_PATH.to_string());
+    settings.insert(CONFIG_WALLET_BACKUP_KEY.to_string(), UNINITIALIZED_BACKUP_KEY.to_string());
 
     error::SUCCESS.code_num
 }
 
 pub fn validate_config(config: &HashMap<String, String>) -> Result<u32, u32> {
 
+    //Mandatory parameters
```

Comment:
This causes tests
test_process_config_str
test_process_file

To fail when run independently.  When run as part of the whole test suite, they pass.

Reply:
I'm working on fixing these.